### PR TITLE
💄Training Pages - Adding Banner and  h1

### LINKED ISSUE #1398: 🧑‍🦼 Events template - Needs a h1
cc @jaydenalchin @hveraus @sethdailyssw @tiagov8 

Every page needs a h1 as per https://www.ssw.com.au/rules/make-title-h1-and-h2-tags-descriptive/

This is important for accessibility and SEO. See WCAG 2.4.6
https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-descriptive.html

WCAG 1.4.5 is also relevant
https://www.w3.org/WAI/WCAG21/Understanding/images-of-text.html

```[tasklist]
### Tasks
- [x] Add a h1 on top of the banner. This should be centered and ~black or white~ sswRed. H1 styling should be consistent with SSW Rules
- [x] Checked by @jaydenalchin
- [x] Check any other pages for the same issue
```

### PR DESCRIPTION
**Pull request by amankumarrr**
- Affected routes: `/events/*`

- Fixed #1398 

- [ ] Include done video or screenshots

MicrosoftTeams-image (9)

**Figure: Angular Workshop page with new h1 and left aligned text**

### CODE REVIEW

File: content/events/angular-workshop.mdx [hunk 1]
```diff
@@ -8,7 +8,15 @@ eventHeader:
   heroBackground: /images/background/polygonBackground.png
   altText: 'Angular Workshop '
   imgOverlay: /images/background/Angular-Workshop.png
-footer: ''
+title: <span class="text-sswRed">Angular Hands on 2-day Workshop</span>
```
Comment (jaydenalchin) [inline content/events/angular-workshop.mdx]: These spans nullify the h1 tag 

1. Please fix! (style the h1 on these pages SSW Red without span)

**Review by jaydenalchin — CHANGES_REQUESTED** [review]

File: content/events/angular-workshop.mdx (hunk 1 above)
Comment (amankumarrr) [reply content/events/angular-workshop.mdx]: fixed, thanks

**Review by jaydenalchin — APPROVED** [review]

**Comment by wicksipedia** [comment]
Test pass on:

- Angular Superpowers
- Angular 2 day workshop
- Azure Superpowers
- Azure Superskills

**Comment by william-liebenberg** [comment]
Test pass on:

.NET Dev Superpowers Tour

**Review by danielmackay — APPROVED** [review]
Clean Architecture changes look good.

**Review by bradystroud — APPROVED** [review]
Looks good!

**Pull request merged**